src/model/AirLoopHVACUnitarySystem.hpp

```diff
@@ -106,7 +108,7 @@ namespace model {
     boost::optional<HVACComponent> supplementalHeatingCoil() const;
 
     /** In EnergyPlus 8.3.0 and above this property maps to the EnergyPlus field "Cooling Supply Air Flow Rate Method" **/
-    boost::optional<std::string> supplyAirFlowRateMethodDuringCoolingOperation() const;
+    std::string supplyAirFlowRateMethodDuringCoolingOperation() const;
```

Comment:
Can we capture this API change for the release notes? @DavidGoldwasser or @tijcolem do we have a place to keep track of these so we don't miss them at release time?

Reply:
Capturing API changes in release notes here: https://github.com/NREL/OpenStudio/pull/4873/commits/52b5d438a7753304037628bdef6a3b69f7147ba2